Repositories/RepositoryHelper.cs

```diff
@@ -34,5 +32,17 @@ public static List<T> LoadFromFile<T>(string filePath, Func<string[], T> parseFu
             }
             return fileData;
         }
+
+        public static void DeleteAllEntireData(string filePath)
+        {
+            if (!File.Exists(filePath))
+            {
+                throw new Exception($"{filePath} does not exist.");
```

Comment:
let's create sub-types for our exceptions for this project and use them, this is a good thing to do so in case we want to catch them and handle them, or if we integrate with FE and we want them to handle these exceptions in a certain way 
and it's not recommended/encouraged to throw `SystemException`  for user code